shell/src/main/java/alluxio/cli/fs/command/MountCommand.java

```diff
@@ -102,14 +115,24 @@ public int run(CommandLine cl) throws AlluxioException, IOException {
       Properties properties = cl.getOptionProperties(OPTION_OPTION.getLongOpt());
       optionsBuilder.putAllProperties(Maps.fromProperties(properties));
     }
-    mFileSystem.mount(alluxioPath, ufsPath, optionsBuilder.build());
+    if (cl.hasOption(VERBOSE_OPTION.getOpt())) {
+      optionsBuilder.setDetail(true);
+      List<String> mountDetailInfos =
```

Comment:
If the command line option is named `verbose`, should make the property `verbosity` to avoid confusion. No need to introduce a new and vague name.

Reply:
done